src/features/Main/Header/index.jsx

```diff
@@ -1,60 +1,46 @@
-import React, { useContext, useState } from 'react';
+import React, { useContext } from 'react';
 import { AppContext } from '@edx/frontend-platform/react';
 import { getConfig } from '@edx/frontend-platform';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faCaretDown } from '@fortawesome/free-solid-svg-icons';
-import { InstitutionContext } from 'features/Main/institutionContext';
+import { faEarthAmericas } from '@fortawesome/free-solid-svg-icons';
+import { Dropdown } from '@edx/paragon';
 import 'features/Main/Header/index.scss';
 
 export const Header = () => {
-  const dataInstitution = useContext(InstitutionContext);
-  const [isAccountMenuOpen, setIsAccountMenuOpen] = useState(false);
-  const handleAccountMenuToggle = () => {
-    setIsAccountMenuOpen((prevState) => !prevState);
-  };
   const { authenticatedUser, config } = useContext(AppContext);
-  const usernameFirstLetter = authenticatedUser.username.charAt(0).toUpperCase();
+  const userName = authenticatedUser.username;
 
   return (
     <header className="institution-header py-4 px-3">
-      <div className="institution-name">
-        {dataInstitution.length > 1 ? (
-          <h3>Global Admin</h3>
-        ) : (
-          <h3>{dataInstitution.length === 1 ? dataInstitution[0].name : 'No Institution Found'}</h3>
-        )}
+      <div className="header-left d-flex">
+        <a href={`${config.LMS_BASE_URL}`}>
```

Comment:
What's the difference between getConfig and config? This should be site-aware. @AuraAlba

Reply:
Fixed, I changed all for getConfig